### PR TITLE
improve show of GroupedDataFrame

### PR DESCRIPTION
**Pull request by bkamins**
Use `repr` to show grouping column values and improve escaping in HTML and LaTeX output.

### CODE REVIEW

File: src/abstractdataframe/io.jl [hunk 1]
```diff
@@ -168,15 +168,15 @@ function Base.show(io::IO, mime::MIME"text/html", gd::GroupedDataFrame)
...
+    keys = html_escape(join(':' .* string.(keynames), ", "))
     keystr = length(gd.cols) > 1 ? "keys" : "key"
     groupstr = N > 1 ? "groups" : "group"
     write(io, "<p><b>$(typeof(gd).name) with $N $groupstr based on $keystr: $keys</b></p>")
     if N > 0
         nrows = size(gd[1], 1)
         rows = nrows > 1 ? "rows" : "row"
 
-        identified_groups = [':' * string(parent_names[col], " = ", first(gd[1][col]))
+        identified_groups = [html_escape(':' * string(parent_names[col], " = ", repr(first(gd[1][col]))))
```
Comment (nalimilan) [inline src/abstractdataframe/io.jl]: While we're at it, I wonder whether we should drop the leading `:` (we don't show it in the column header). Cc: @pdeffebach

Comment (bkamins) [reply src/abstractdataframe/io.jl]: I mildly lean towards dropping `:`, but I am OK with both.

**Comment by bkamins** [comment]
A side question: do you know why AppVeyor shows that it has failed on nightly x64, while the log shows that the tests have passed?

File: src/abstractdataframe/io.jl (hunk 1 above)
Comment (pdeffebach) [reply src/abstractdataframe/io.jl]: Yeah I do, too. I think we should drop the `:` for consistency.

Comment (bkamins) [reply src/abstractdataframe/io.jl]: OK - I will drop it in this PR.

**Pull request merged**